radarr/faq.md

```diff
@@ -522,7 +522,7 @@ Depending on your OS, there are multiple possible ways.
 
 - Radarr uses [The Movie Database (TMDb)](http://themoviedb.org) for movie information and images like fanart, banners and backgrounds. Generally, there are a few reasons why you may not be able to add a movie:
   - TMDb doesn't like special characters to be used when searching for movies through the API (which Radarr uses), so try searching a translated name, and/or without special characters.
-  - You can also add by TMDb ID or, if TMDb has it, the IMDb ID
+  - You can also add by TMDb ID or, if TMDb does not have it, the IMDb ID
```

Comment:
If tmdb does not have the imdbid you cannot use the imdbid to add the movie.

If the movie is not on tmdb it cannot be added

Reply:
Oh, so it was correct. Then we can close this :)